app/config/FrontendAppConfig.scala

```diff
@@ -121,4 +121,6 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
 
   def routeToSwitchLanguage: String => Call =
     (lang: String) => routes.LanguageSwitchController.switchToLanguage(lang)
+
+  lazy val srnRegex: String = "^S[0-9]{10}$"
```

Comment:
Not sure whether this is the right place for this - any particular reason why you want to put it here?

Reply:
It's used in both ChargeDetailsController and PenaltiesController: `if (identifier.matches(appConfig.srnRegex))`

Reply:
Well the app config object is usually used for reading stuff from the app config file, so unless you are wanting to make the regex configurable it might be better to put the regex elsewhere in the code. Look in scheme fe - I think they have a Constants file there where they store regexes.

Reply:
Moved to constants file, reused throughout.